modular-docs-manual/content/topics/module_anchor-and-file-names-concept.adoc

```diff
@@ -0,0 +1,52 @@
+[#anchor-and-file-names]
+= Anchor Names and File Names
+
+To optimize modular documentation, follow these guidelines for naming module anchors and files.
+
+Anchor names:: Provide an anchor for every module so that it can be identified when reused or cross-referenced. Give the anchor name the same or a similar name to the module heading, separated by underscores, and indicate the module type (`con`, `ref`, `proc`) at the end of the name. If applicable, consider also providing a module sub-type (`create`, `edit`, `delete`) before the module type:
```

Comment:
I would appreciate elaborating in the parentheses about the module types, but that's just a personal preference:
```
...indicate the module type (`con` for concept, `ref` for reference, and `proc` for procedure modules) at the end...
```

Reply:
Agreed. Done.